database/sqlite/sqlite_aclk_alert.c

```diff
@@ -344,6 +344,10 @@ void aclk_push_alert_event(struct aclk_database_worker_config *wc, struct aclk_d
                                       strdupz((char *)"") :
                                       strdupz((char *)sqlite3_column_text(res, 18));
 
+        alarm_log.chart_context = sqlite3_column_type(res, 26) == SQLITE_NULL ?
```

Comment:
This variable is not cleaned after `aclk_send_alarm_log_entry` call. Is there a specific reason for this?

Reply:
No! Good catch, thanks!! Added.